ru/mail/chrome/messenger/pbMessenger.dtd

```diff
@@ -363,14 +363,14 @@
  entities below were automatically added by l10nmerge
 -->
 
-<!ENTITY pbContentTextContextMenu-evernote.label       "Add to Evernote">
-<!ENTITY fullScreenMode.label "Enter Full Screen">
-<!ENTITY pbMessagePaneContext-evernote.label     "Add to Evernote">
+<!ENTITY pbContentTextContextMenu-evernote.label       "Добавить в Evernote">
+<!ENTITY fullScreenMode.label "Перейти в полноэкранный режим">
+<!ENTITY pbMessagePaneContext-evernote.label     "Добавить в Evernote">
 <!ENTITY pbContentTextContextMenu-date.accesskey        "d">
 <!ENTITY fullScreenMode.accesskey "l">
-<!ENTITY evernoteButton.tooltip "Add to Evernote">
+<!ENTITY evernoteButton.tooltip "Добавить в Evernote">
 <!ENTITY pbMessagePaneContext-evernote.accesskey "e">
-<!ENTITY pbContentTextContextMenu-date.label            "Add to Calendar">
+<!ENTITY pbContentTextContextMenu-date.label            "Добавить в календарь">
```

Comment:
CORRECTIONS NEEDED for line 373:
+<!ENTITY pbContentTextContextMenu-date.label            "Добавить в Календарь">
It's about integration with Google Calendar, I guess, since source line contains a capitalised "Calendar" term.